sql/core/src/test/scala/org/apache/spark/sql/execution/GlobalTempViewSuite.scala

```diff
@@ -170,4 +170,14 @@ class GlobalTempViewSuite extends QueryTest with SharedSQLContext {
         isTemporary = true).toString)
     }
   }
+
+  test("Create global temp view with duplicate column name") {
+    withGlobalTempView("src") {
+      val e = intercept[AnalysisException]{
+        sql("CREATE GLOBAL TEMP VIEW src AS SELECT 1 as a, 2 as a")
+      }
+      assert(e.getMessage.contains("Found duplicate column(s)"))
+      sql("CREATE GLOBAL TEMP VIEW src AS SELECT 1 as a, 2 as b")
+    }
+  }
```

Comment:
Could you update `org.apache.spark.sql.hive.ErrorPositionSuite`, too?

Reply:
Fixed.